src/server/system_services/schemas/nsfs_config_schema.js

```diff
@@ -54,7 +54,17 @@ const nsfs_node_config_schema = {
         GPFS_DL_PATH: {
             type: 'string',
             default: '',
-            description: 'indicate the location of the gpfs library file, service restart required, usually should be set to /usr/lib64/libgpfs.so'
+            description: 'indicates the location of the gpfs library file, service restart required, usually should be set to /usr/lib64/libgpfs.so'
+        },
+        NSFS_NC_STORAGE_BACKEND: {
+            $ref: 'common_api#/definitions/fs_backend',
+            default: '',
```

Comment:
This default does not match the schema (it is a string that is not part of the enum), might be confusing.

Reply:
np I changed it to undefined